unit_tests/sources/declarative/extractors/test_record_filter.py

```diff
@@ -56,7 +56,7 @@
     "filter_template, records, expected_records",
     [
         (
-            "{{ record['created_at'] > stream_state['created_at'] }}",
+            "{{ record['created_at'] >= stream_interval.extra_fields['created_at'] }}",
```

Comment:
this is an interesting test. Not an issue w/ your change, but its interesting that this test during setup injects a `created_at` into extra values slices